buildSrc/src/main/kotlin/com/saveourtool/save/buildutils/SystemUtils.kt

```diff
@@ -0,0 +1,26 @@
+/**
+ * Utility functions for System
+ */
+
+package com.saveourtool.save.buildutils
+
+private fun String.isWindows() = startsWith("Windows", ignoreCase = true)
```

Comment:
It's better to use Gradle's `DefaultNativePlatform` and `DefaultOperatingSystem` here.